src/img/icons/advanced.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36"><circle fill="#FFCC4D" cx="18" cy="18" r="18"/><path fill="#664500" d="M27.335 23.629c-.178-.161-.444-.171-.635-.029-.039.029-3.922 2.9-8.7 2.9-4.766 0-8.662-2.871-8.7-2.9-.191-.142-.457-.13-.635.029-.177.16-.217.424-.094.628C8.7 24.472 11.788 29.5 18 29.5s9.301-5.028 9.429-5.243c.123-.205.084-.468-.094-.628z"/><path fill="#65471B" d="M18 26.591c-.148 0-.291-.011-.438-.016v4.516h.875v-4.517c-.145.005-.289.017-.437.017z"/><path fill="#FFF" d="M22 26c.016-.004-1.45.378-2.446.486-.366.042-.737.076-1.117.089v4.517H20c1.1 0 2-.9 2-2V26zm-8 0c-.016-.004 1.45.378 2.446.486.366.042.737.076 1.117.089v4.517H16c-1.1 0-2-.9-2-2V26z"/><path fill="#65471B" d="M27.335 23.629c-.178-.161-.444-.171-.635-.029-.03.022-2.259 1.668-5.411 2.47-.443.113-1.864.43-3.286.431-1.424 0-2.849-.318-3.292-.431-3.152-.802-5.381-2.448-5.411-2.47-.19-.142-.457-.132-.635.029-.178.16-.217.423-.094.628.097.162 1.885 3.067 5.429 4.481v-1.829c-.016-.004 1.45.378 2.446.486.366.042.737.076 1.117.089.146.005.289.016.437.016.148 0 .291-.011.438-.016.38-.013.751-.046 1.117-.089.996-.108 2.462-.49 2.446-.486v1.829c3.544-1.414 5.332-4.319 5.429-4.481.122-.205.083-.468-.095-.628zm-.711-9.605c0 1.714-.938 3.104-2.096 3.104-1.157 0-2.096-1.39-2.096-3.104s.938-3.104 2.096-3.104c1.158 0 2.096 1.39 2.096 3.104zm-17.167 0c0 1.714.938 3.104 2.096 3.104 1.157 0 2.096-1.39 2.096-3.104s-.938-3.104-2.096-3.104c-1.158 0-2.096 1.39-2.096 3.104z"/><path fill="#292F33" d="M34.808 9.627c-.171-.166-1.267.274-2.376-.291-2.288-1.166-8.07-2.291-11.834.376-.403.285-2.087.333-2.558.313-.471.021-2.155-.027-2.558-.313-3.763-2.667-9.545-1.542-11.833-.376-1.109.565-2.205.125-2.376.291-.247.239-.247 1.196.001 1.436.246.239 1.477.515 1.722 1.232.247.718.249 4.958 2.213 6.424 1.839 1.372 6.129 1.785 8.848.238 2.372-1.349 2.289-4.189 2.724-5.881.155-.603.592-.907 1.26-.907s1.105.304 1.26.907c.435 1.691.351 4.532 2.724 5.881 2.719 1.546 7.009 1.133 8.847-.238 1.965-1.465 1.967-5.706 2.213-6.424.245-.717 1.476-.994 1.722-1.232.248-.24.249-1.197.001-1.436zm-20.194 3.65c-.077 1.105-.274 3.227-1.597 3.98-.811.462-1.868.743-2.974.743h-.001c-1.225 0-2.923-.347-3.587-.842-.83-.619-1.146-3.167-1.265-4.12-.076-.607-.28-2.09.388-2.318 1.06-.361 2.539-.643 4.052-.643.693 0 3.021.043 4.155.741 1.005.617.872 1.851.829 2.459zm16.278-.253c-.119.954-.435 3.515-1.265 4.134-.664.495-2.362.842-3.587.842h-.001c-1.107 0-2.163-.281-2.975-.743-1.323-.752-1.52-2.861-1.597-3.966-.042-.608-.176-1.851.829-2.468 1.135-.698 3.462-.746 4.155-.746 1.513 0 2.991.277 4.052.638.668.228.465 1.702.389 2.309z"/></svg>
```

Comment:
This is a really badly compressed `svg` image. This is not your fault, it's just the image that the emoji library supplied.

Let me try to make it better.

Reply:
Here, replace it with this:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg viewBox="0 0 30 30"><path d="M 30,15 A 15,15 0 0 1 15,30 15,15 0 0 1 0,15 15,15 0 0 1 15,0 15,15 0 0 1 30,15 Z" style="fill:#ffcc4d;stroke-width:0.83333331" /><path style="fill:#664500;stroke-width:0.83333331" d="m 22.779167,19.690833 c -0.148334,-0.134166 -0.37,-0.1425 -0.529167,-0.02417 -0.0325,0.02417 -3.268333,2.416666 -7.25,2.416666 -3.971667,0 -7.2183333,-2.3925 -7.25,-2.416666 -0.1591667,-0.11833 -0.3808333,-0.10833 -0.5291667,0.02417 C 7.0733333,19.824167 7.04,20.044167 7.1425,20.214167 7.25,20.393333 9.8233333,24.583333 15,24.583333 c 5.176667,0 7.750833,-4.19 7.8575,-4.369166 0.1025,-0.170834 0.07,-0.39 -0.07833,-0.523334 z" /><path style="fill:#65471b;stroke-width:0.83333331" d="m 15,22.159167 c -0.123333,0 -0.2425,-0.0092 -0.365,-0.01333 v 3.763334 h 0.729167 V 22.145 C 15.243333,22.1492 15.123333,22.15917 15,22.15917 Z" /><path style="fill:#ffffff;stroke-width:0.83333331" d="m 18.333333,21.666667 c 0.01333,-0.0033 -1.208333,0.315 -2.038333,0.405 -0.305,0.035 -0.614167,0.06333 -0.930833,0.07417 V 25.91 h 1.3025 c 0.916666,0 1.666666,-0.75 1.666666,-1.666667 z m -6.666666,0 c -0.01333,-0.0033 1.208333,0.315 2.038333,0.405 0.305,0.035 0.614167,0.06333 0.930833,0.07417 V 25.91 h -1.3025 c -0.916666,0 -1.666666,-0.75 -1.666666,-1.666667 z" /><path style="fill:#65471b;stroke-width:0.83333331" d="m 22.779167,19.690833 c -0.148334,-0.134166 -0.37,-0.1425 -0.529167,-0.02417 -0.025,0.01833 -1.8825,1.39 -4.509167,2.058333 -0.369166,0.09417 -1.553333,0.358333 -2.738333,0.359167 -1.186667,0 -2.374167,-0.265 -2.743333,-0.359167 C 9.6325,21.056667 7.775,19.685 7.75,19.666667 7.5916667,19.548333 7.3691667,19.556667 7.2208333,19.690833 7.0725,19.824167 7.04,20.043333 7.1425,20.214167 c 0.080833,0.135 1.5708333,2.555833 4.524167,3.734166 v -1.524166 c -0.01333,-0.0033 1.208333,0.315 2.038333,0.405 0.305,0.035 0.614167,0.06333 0.930833,0.07417 0.121667,0.0042 0.240834,0.01333 0.364167,0.01333 0.123333,0 0.2425,-0.0092 0.365,-0.01333 0.316667,-0.01083 0.625833,-0.03833 0.930833,-0.07417 0.83,-0.09 2.051667,-0.408334 2.038334,-0.405 v 1.524166 C 21.2875,22.77 22.7775,20.349167 22.858333,20.214167 c 0.101667,-0.170834 0.06917,-0.39 -0.07917,-0.523334 z m -0.5925,-8.004166 c 0,1.428333 -0.781667,2.586666 -1.746667,2.586666 -0.964167,0 -1.746667,-1.158333 -1.746667,-2.586666 C 18.693333,10.258333 19.475,9.1 20.44,9.1 c 0.965,0 1.746667,1.158333 1.746667,2.586667 z m -14.3058337,0 c 0,1.428333 0.7816667,2.586666 1.7466667,2.586666 0.964167,0 1.746667,-1.158333 1.746667,-2.586666 C 11.374167,10.258333 10.5925,9.1 9.6275,9.1 c -0.965,0 -1.7466667,1.158333 -1.7466667,2.586667 z" /><path style="fill:#292f33;stroke-width:0.83333331" d="M 29.006667,8.0225 C 28.864167,7.8841667 27.950833,8.2508333 27.026667,7.78 25.12,6.8083333 20.301667,5.8708333 17.165,8.0933333 c -0.335833,0.2375 -1.739167,0.2775 -2.131667,0.2608334 -0.3925,0.0175 -1.795833,-0.0225 -2.131666,-0.2608334 C 9.7658333,5.8708333 4.9475,6.8083333 3.0408333,7.78 c -0.9241666,0.4708333 -1.8375,0.1041667 -1.98,0.2425 C 0.855,8.2216667 0.855,9.0191667 1.0616667,9.2191667 c 0.205,0.1991666 1.2308333,0.4291666 1.435,1.0266663 0.2058333,0.598334 0.2075,4.131667 1.8441666,5.353334 1.5325,1.143333 5.1075,1.4875 7.3733337,0.198333 1.976666,-1.124167 1.9075,-3.490833 2.27,-4.900833 0.129166,-0.5025 0.493333,-0.755834 1.05,-0.755834 0.556666,0 0.920833,0.253334 1.05,0.755834 0.3625,1.409166 0.2925,3.776666 2.27,4.900833 2.265833,1.288333 5.840833,0.944167 7.3725,-0.198333 1.6375,-1.220834 1.639166,-4.755 1.844166,-5.353334 0.204167,-0.5974997 1.23,-0.828333 1.435,-1.0266663 0.206667,-0.2 0.2075,-0.9975 8.34e-4,-1.1966667 z M 12.178333,11.064167 C 12.114167,11.985 11.95,13.753333 10.8475,14.380833 10.171667,14.765833 9.2908333,15 8.3691667,15 H 8.3683333 C 7.3475,15 5.9325,14.710833 5.3791667,14.298333 4.6875,13.7825 4.4241667,11.659167 4.325,10.865 4.2616667,10.359167 4.0916667,9.1233333 4.6483333,8.9333333 5.5316667,8.6325 6.7641667,8.3975 8.025,8.3975 c 0.5775,0 2.5175,0.035833 3.4625,0.6175 0.8375,0.5141667 0.726667,1.5425 0.690833,2.049167 z m 13.565,-0.210834 c -0.09917,0.795 -0.3625,2.929167 -1.054166,3.445 C 24.135833,14.710833 22.720833,15 21.7,15 h -8.33e-4 c -0.9225,0 -1.8025,-0.234167 -2.479167,-0.619167 -1.1025,-0.626666 -1.266667,-2.384166 -1.330833,-3.305 C 17.854167,10.569167 17.7425,9.5333333 18.58,9.0191667 19.525833,8.4375 21.465,8.3975 22.0425,8.3975 c 1.260833,0 2.4925,0.2308333 3.376667,0.5316667 0.556666,0.19 0.3875,1.4183333 0.324166,1.9241663 z" /></svg>
```